### PR TITLE
Add our take on unsupported mods to the crash report

### PR DESCRIPTION
**Pull request by HenryLoenwind**
Also stop early when AE2 rv1 is loaded

### CODE REVIEW

**Comment by tterrag1098** [comment]
Why not use the @Mod dependencies field?

File: src/main/java/crazypants/enderio/tool/EnderIOCrashCallable.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    if (hasOptifine) {
+      msg += "\"Optifine is installed. This is NOT supported.\" ";
+    }
+    if (hasBadAE2) {
+      msg += "\"AE2 rv1 is installed. Ender IO is no longer compatible with rv1, please update to rv2.\" ";
+    }
+    if (hasUnknownAE2) {
+      msg += "\"An unknwon AE2 API is installed. Ender IO was build against API version rv2.\" ";
+    }
+    msg += "This may have caused the error. Try reproducing the crash WITHOUT this/these mods before reportin it.";
```
Comment (Vexatos) [inline src/main/java/crazypants/enderio/tool/EnderIOCrashCallable.java]: "reportin"........

**Comment by HenryLoenwind** [comment]
Ugh. Never directly PR a change that was done in one sitting after a very long day at work...

Sorry, quality was bad, feature set was bad. This version should be more sensible in both.

**Comment by tterrag1098** [comment]
I'm ok with the Opifine bit, but AE2 is no longer needed.

**Comment by HenryLoenwind** [comment]
I already removed the rv1 check...

**Comment by CrazyPants** [comment]
Is optifine really needed? I havent been around for a bit but I cant remeber any crash bugs being reported that where caused by optifine. Any issues I have seen with it where rendering related. I am not 100% sure if it these even effected eio.

**Comment by HenryLoenwind** [comment]
We had some "already tessellating" ones, I think.

But this PR is more or less becoming a "add the class so we can add messages later if we need to"...

**Pull request merged**